specification/framework.md

```diff
@@ -264,7 +264,7 @@ Depending on the projects and their criteria, you may have a mix of framework le
 
 **Level 2** – This maturity level focuses on shifting security further left by improving ingestion configuration security, decreasing MTTR to patch OSS vulnerabilities, and responding to incidents. The SaltStack vulnerability in 2020 showed us that adversaries were able to start exploiting CVE-2020-11651 within 3 days of it being announced. Even though a patch was available, organizations were not able to patch their systems fast enough. Thus, a key component of this level leverages automation to help developers keep their OSS hygiene healthy and updated. The ideal goal is for organizations to be able to patch faster than attackers can operate.
 
-**Level 3** – Proactively performing security analysis on your organization&#39;s most used OSS components and reducing risk to consume malicious packages are the themes of this maturity level. Scanning for malware in OSS before the package is downloaded is key toward preventing compromise. Then, to perform proactive security reviews of OSS requires that an organization can clone the source code to an internal location. Proactive security reviews help you look for the not-yet-discovered vulnerabilities, as well as identifying other threat categories such as detecting backdoors.
+**Level 3** – Proactively performing security analysis on your organization&#39;s most used OSS components and reducing risk to consume malicious packages are the themes of this maturity level. Scanning for malware in OSS before the package is downloaded is key toward preventing compromise. Then, to perform proactive source code analysis of OSS requires that an organization can clone the source code to an internal location. Proactive security analysis help you look for the not-yet-discovered vulnerabilities, as well as identifying other threat categories such as detecting backdoors.
```

Comment:
Non-blocking feedback: I can support either proactive "security analysis" or "source code analysis," but think a separate issue and a follow-on change would itemize one or more examples that are exemplary and relevant to engineers or engineering managers.

Reply:
Thanks for the feedback, both. Updated in 26792f785223cc5bef7ebe08fa82e4323e951072